ginga/misc/plugins/Cuts.py

```diff
@@ -221,6 +235,29 @@ def build_gui(self, container):
         self.select_cut(self.cutstag)
         self.gui_up = True
 
+    def build_axes(self, vbox):
+        fr = Widgets.Frame("Axes")
+        hbox = Widgets.HBox()
+        hbox.set_border_width(4)
+        hbox.set_spacing(1)
+
+        image = self.fitsimage.get_image()
+        if image is not None:
+            # Add Checkbox widgets
+            for i in xrange(0, len(image.get_mddata().shape)+1):
+                name = 'NAXIS%d' % i
+                chkbox = Widgets.CheckBox(name)
+                self.axes_chkboxes.append(Bunch.Bunch(widget=chkbox, pos=i, state=False))
+                hbox.add_widget(chkbox)
+
+            # Add callbacks
+            for chkbox in self.axes_chkboxes:
+                cbox = chkbox.get('widget')
+                cbox.add_callback('activated', lambda w, tf: self.axis_toggle_cb(w, tf, chkbox))
```

Comment:
@rupak0577, are you trying to add radio button like behavior with exclusivity?  If so, please use the `Widgets.RadioButton`.  For second and remaining buttons you specify the "group" using the group parameter which refers to the first widget in the group.  Also, you *may* get a callback with `False` when the widget has toggled "off" so you want to check for True before you actually do something.  You can see examples in the `Cuts` plugin radio button processing.